liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -176,15 +175,18 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 	}
 }
 
-void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
+void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg, bool jsonOutput)
```

Comment:
```suggestion
void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg, bool _jsonOutput)
```

Also in a few other places.

Reply:
Also, I think this could have a more generic name since it does not have to be tied to JSON in general (even if now it is). For example:
```suggestion
void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg, bool _printFullType)
```